src/UITests/app.config

```diff
@@ -6,6 +6,22 @@
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
>dependentAssembly [](start = 7, length = 17)

Changes in this file are not related to the Axe.Windows change, but VS made the updates anyway. Happy to remove these changes from this PR if desired.